README.md

```diff
@@ -92,4 +92,4 @@ help .\build.ps1 -Detailed
 ```
 
 > Note: If you don't specify `-Version <version>` the script will try to infer it from tags pointing to the current git commit. An error is produced if you don't have a tag and no version is provided.
-| See code examples. Do a coding tutorial. Watch guest lectures.          | Get design guides. Build user interface. Learn interactions and input.     | Get development guides. Learn the technology. Understand the science.       | Join open source projects. Ask questions on forums. Attend events and meet-ups. |
\ No newline at end of file
+| See code examples. Do a coding tutorial. Watch guest lectures.          | Get design guides. Build user interface. Learn interactions and input.     | Get development guides. Learn the technology. Understand the science.       | Join open source projects. Ask questions on forums. Attend events and meet-ups. |
```

Comment:
This part looks like it gets caught in the note:

![image](https://user-images.githubusercontent.com/3580640/53041515-0b5dfd00-3439-11e9-845b-e9a0acf0790f.png)

Should we remove it? Or I guess at least add another return to get it out of the scope of the `>`

Reply:
Does anyone know what these different | stubs are for? They don't hyperlink to anything so I would probably suggest removing them?

Reply:
I'm down for that